src/react/AppProvider.jsx

```diff
@@ -1,7 +1,8 @@
-import React, { useState, useMemo } from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { Router } from 'react-router-dom';
 
+import { sendTrackEvent } from '../analytics/index';
```

Comment:
nit: is the `/index` necessary? Importing from `../analytics` should suffice as it'll pull from index already.

Reply:
done